test/FormElements.test.ts

```diff
@@ -22,18 +23,20 @@ async function createForm(): Promise<HTMLFormElement> {
       <test-input name="second" id="ti2"></test-input>
       <test-input name="third" id="ti3"></test-input>
       <button type="submit">Submit</button>
+      <md-filled-button type="submit">Submit Material</md-filled-button>
```

Comment:
Drive-by comment, I'd create a standalone test class instead of importing MWC

```ts
class TestFormAssociatedNoAttachInternals extends HTMLElement {
  static formAssociated = true;
}
```